roles/tpa_single_node/templates/manifests/guac/bombastic-collector/Deployment.yaml.j2

```diff
@@ -37,45 +38,52 @@ spec:
             - --poll
           env:
             - name: STORAGE_ACCESS_KEY
```

Comment:
```suggestion
            - name: STORAGE_ACCESS_KEY
              valueFrom:
                secretKeyRef:
                  key: storage_access_key
                  name: storage_secret
            - name: STORAGE_SECRET_KEY
              valueFrom:
                secretKeyRef:
                  key: storage_secret_key
                  name: storage_secret

{% if 'minio' in tpa_single_node_storage_type %}   
            - name: GUAC_S3_URL
              value: "{{ tpa_single_node_storage_endpoint }}"
{% elif 's3' in tpa_single_node_storage_type %}
            - name: STORAGE_REGION
              value: "{{ tpa_single_node_storage_region }}"
{% endif %}

            - name: GUAC_S3_BUCKET
              value: "{{ tpa_single_node_storage_bombastic_bucket }}"
            - name: GUAC_S3_MP
              value: "{{ tpa_single_node_event_bus_type }}"
{% if 'sqs' in tpa_single_node_event_bus_type %}
            - name: SQS_ACCESS_KEY
              valueFrom:
                secretKeyRef:
                  key: event_access_key_id
                  name: event_secret
            - name: SQS_SECRET_KEY
              valueFrom:
                secretKeyRef:
                  key: event_secret_access_key
                  name: event_secret
            - name: SQS_REGION
              value: "{{ tpa_single_node_sqs_region }}"
{% elif 'kafka' in tpa_single_node_event_bus_type %}
            - name: GUAC_S3_MP_ENDPOINT
              value: "{{ tpa_single_node_kafka_bootstrap_servers }}"
            - name: KAFKA_PROPERTIES_ENV_PREFIX
              value: TCK_
            - name: TCK_SECURITY__PROTOCOL
              value: "{{ tpa_single_node_kafka_security_protocol }}"
            - name: TCK_SASL__USERNAME
              valueFrom:
                secretKeyRef:
                  key: event_access_key_id
                  name: event_secret
            - name: TCK_SASL__PASSWORD
              valueFrom:
                secretKeyRef:
                  key: event_secret_access_key
                  name: event_secret
            - name: TCK_SASL__MECHANISM
              value: "{{ tpa_single_node_kafka_auth_mechanism }}"
{% endif %}
            - name: GUAC_S3_QUEUES
              value: "{{ tpa_single_node_bombastic_topic_indexed }}"
            - name: GUAC_GQL_ADDR
              value: 127.0.0.1:8080/query
            - name: GUAC_CSUB_ADDR
              value: 127.0.0.1:2782
          volumeMounts:
            - mountPath: /etc/bombasticsecret
              name: bombasticsecret
            - mountPath: /etc/bombasticsqssecret
              name: bombasticsqssecret
        - name: bombasticsecret
          secret:
            secretName: storage_secret
        - name: bombasticsqssecret
          secret:
            secretName: event_secret
```

Reply:
I see some misalignment with the conditional statements, for example SQS_ACCESS_KEY and SQS_SECRET_KEY being included with tpa_single_node_storage_type conditional block.
And the current template leverages kafka fields which might not be relevant for AWS SQS.

I have added a suggestion for bombastic-collector deployment template and I hope the same one can be used for vexination-collector.